config.php.default

```diff
@@ -49,3 +49,6 @@ $config['enable_billing'] = 1;
 
 # Enable the in-built services support (Nagios plugins)
 $config['show_services'] = 1;
+
+# Uncomment the following line if you're using HTTPS and want secure session cookies
+# ini_set('session.cookie_secure', 1);
```

Comment:
Don't put code in this file, only config definitions.
You can move this to init.php